### PR TITLE
Himanshu Blog May 16

### PR DESCRIPTION
**Pull request by hghimanshu**

### CODE REVIEW

**Comment by Rajeev111974** [comment]
Hi himanshu,

 

Please check the blog and update the content as per mentioned comment in github.

 

With Regards

Rajeev Kumar

Operations Manager, 

Nayan India Science & Technologies (P) Ltd.

+919958083303

rajeev@nayantech.com

 

This e-mail and any attachments contain information from NAYAN Companies and are intended solely for the use of the named recipient or recipients. This e-mail may contain confidential and proprietary information. Any dissemination of this e-mail by anyone other than an intended recipient is strictly prohibited. If you are not a named recipient, you are prohibited from any further viewing of the e-mail or any attachments or from making any use of the e-mail or attachments. If you believe you have received this e-mail in error, notify the sender immediately and permanently delete the e-mail, any attachments, and all copies thereof from any drives or storage media and destroy any printouts of the e-mail or attachments.

**Comment by Rajeev111974** [comment]
HI Himanshu,

 

We have checked he blog and still changes is there for update, Please check and update it as suggestion has been mentioned there.

 

https://github.com/nayan-tech/nayan-blog/pull/43

 

With Regards

Rajeev Kumar

Operations Manager, 

Nayan India Science & Technologies (P) Ltd.

+919958083303

rajeev@nayantech.com

 

This e-mail and any attachments contain information from NAYAN Companies and are intended solely for the use of the named recipient or recipients. This e-mail may contain confidential and proprietary information. Any dissemination of this e-mail by anyone other than an intended recipient is strictly prohibited. If you are not a named recipient, you are prohibited from any further viewing of the e-mail or any attachments or from making any use of the e-mail or attachments. If you believe you have received this e-mail in error, notify the sender immediately and permanently delete the e-mail, any attachments, and all copies thereof from any drives or storage media and destroy any printouts of the e-mail or attachments.

 

From: Rajeev Kumar [mailto:rajeev@nayantech.com] 
Sent: Monday, June 15, 2020 4:42 PM
To: 'nayan-tech/nayan-blog'; 'nayan-tech/nayan-blog'; Himanshu Garg
Cc: 'Review requested'; jayant@nayantech.com
Subject: RE: [nayan-tech/nayan-blog] Himanshu Blog May 16 (#43)

 

Hi himanshu,

 

Please check the blog and update the content as per mentioned comment in github.

 

With Regards

Rajeev Kumar

Operations Manager, 

Nayan India Science & Technologies (P) Ltd.

+919958083303

rajeev@nayantech.com

 

This e-mail and any attachments contain information from NAYAN Companies and are intended solely for the use of the named recipient or recipients. This e-mail may contain confidential and proprietary information. Any dissemination of this e-mail by anyone other than an intended recipient is strictly prohibited. If you are not a named recipient, you are prohibited from any further viewing of the e-mail or any attachments or from making any use of the e-mail or attachments. If you believe you have received this e-mail in error, notify the sender immediately and permanently delete the e-mail, any attachments, and all copies thereof from any drives or storage media and destroy any printouts of the e-mail or attachments.

File: source/_posts/Creating-deep-learning-models-in-Docker.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+---
+title: Creating deep-learning models in Docker
```
[the remark this answers is not in the document]
Comment (hghimanshu) [reply source/_posts/Creating-deep-learning-models-in-Docker.md]: updated please check

File: source/_posts/Creating-deep-learning-models-in-Docker.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+date: 2020-05-16 11:09:31
+author: Himanshu Garg
+tags: docker
```
[the remark this answers is not in the document]
Comment (hghimanshu) [reply source/_posts/Creating-deep-learning-models-in-Docker.md]: updated please check

File: source/_posts/Creating-deep-learning-models-in-Docker.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
+---
```
[the remark this answers is not in the document]
Comment (Rajeev111974) [reply source/_posts/Creating-deep-learning-models-in-Docker.md:1]: Done

[the remark this answers is not in the document]
Comment (Rajeev111974) [reply source/_posts/Creating-deep-learning-models-in-Docker.md:1]: Done

[the remark this answers is not in the document]
Comment (Rajeev111974) [reply source/_posts/Creating-deep-learning-models-in-Docker.md:1]: done

**Comment by Rajeev111974** [comment]
Dear Abhishek,

 

Below mentioned blog is reviewed and ready for live, please do the thing needful.

 

With Regards

Rajeev Kumar

Operations Manager, 

Nayan India Science & Technologies (P) Ltd.

+919958083303

rajeev@nayantech.com

 

This e-mail and any attachments contain information from NAYAN Companies and are intended solely for the use of the named recipient or recipients. This e-mail may contain confidential and proprietary information. Any dissemination of this e-mail by anyone other than an intended recipient is strictly prohibited. If you are not a named recipient, you are prohibited from any further viewing of the e-mail or any attachments or from making any use of the e-mail or attachments. If you believe you have received this e-mail in error, notify the sender immediately and permanently delete the e-mail, any attachments, and all copies thereof from any drives or storage media and destroy any printouts of the e-mail or attachments.

**Review by Rajeev111974 — COMMENTED** [review]
Checked and ok for live now

**Pull request merged**